src/lib/stories.ts

```diff
@@ -319,6 +319,7 @@ const printFormattedStory = (program: any) => {
                 )
                 .replace(/%u/, url)
                 .replace(/%a/, `${story.archived}`)
+                .replace(/%gb/, `${buildStoryBranch(story)}`)
```

Comment:
Add: I didn't think about this but having the option to pass in a prefix would be handy on cli